ivy/functional/ivy/losses.py

```diff
@@ -266,3 +266,45 @@ def sparse_cross_entropy(
     """
     true = ivy.one_hot(true, pred.shape[axis])
     return ivy.cross_entropy(true, pred, axis=axis, epsilon=epsilon, out=out)
+
+sparse_cross_entropy.unsupported_dtypes = {"torch": ("float16",)}
```

Comment:
`.unsupported_dtypes` shouldn't be hear, please remove this!

Reply:
Thanks for your review! However, my contribution is smooth_l1_function part, code related to cross_entropy isn't writen by me. Should I still change it?

Reply:
Sorry, then in that case you don't need to worry about it!